### PR TITLE
Play wright user registration Susmita

### PR DESCRIPTION
**Pull request by susmitathapa**
### Description
- In this PR playwright implementation is done in user registration feature

### CODE REVIEW

File: Features5/tests/config.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ BeforeAll(async () => {
   global.browser = await chromium.launch({
     //browser
     headless: false,
-    //slowMo: 200,
+    slowMo: 100,
```
Comment (kiranadh1452) [inline Features5/tests/config.js]: not needed

File: Features5/tests/stepDefinitions/userRegistrationContext.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    const name = myData[i].name;
+    const email = myData[i].email;
+    const password = myData[i].password;
+    const confirmPassword = myData[i].confirmPassword;
+    await page.type(nameSelector, name);
+    await page.type(emailSelector, email);
+    await page.type(passwordSelector, password);
+    await page.type(confirmPasswordSelector, confirmPassword);
+    await page.click(registerSelector);
+    await page.pause();
```
Comment (kiranadh1452) [inline Features5/tests/stepDefinitions/userRegistrationContext.js]: Why pause?

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

File: Features5/tests/stepDefinitions/userRegistrationContext.js [hunk 3]
```diff
@@ -1,22 +1,45 @@
...
+      const name = myData[i].name;
+      const email = myData[i].email;
+      const password = myData[i].password;
+      const confirmPassword = myData[i].confirmPassword;
+      await page.type(nameSelector, name);
+      await page.type(emailSelector, email);
+      await page.type(passwordSelector, password);
+      await page.type(confirmPasswordSelector, confirmPassword);
+      await page.click(registerSelector);
+      // await page.pause();
```
Comment (kiranadh1452) [inline Features5/tests/stepDefinitions/userRegistrationContext.js]:
```suggestion
```

Comment (kiranadh1452) [inline Features5/tests/stepDefinitions/userRegistrationContext.js]: This is repeated. Re-use the code.

File: Features5/tests/stepDefinitions/userRegistrationContext.js [hunk 4]
```diff
@@ -28,11 +51,25 @@ When("user enters following valid credentials:", async function (dataTable) {
...
     await page.type(confirmPasswordSelector, confirmPassword);
     await page.click(registerSelector);
-    await page.pause();
   }
 });
 
+Given('the user has been navigated to homepage', async function () {
+  await page.isVisible(logoutSelector);
+});
+
```
Comment (kiranadh1452) [inline Features5/tests/stepDefinitions/userRegistrationContext.js:59]: Maybe we can skip this step and put this code inside previous step.

Comment (kiranadh1452) [inline Features5/tests/stepDefinitions/userRegistrationContext.js:59]: Maybe we can skip this step and put this code inside previous step.

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

**Review by SagarGi — APPROVED** [review]

**Pull request merged**